CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v10.8.0 - TBD
```

Comment:
Strictly speaking a fix is only a patch version update, but chances are we'll add an enhancement or I'll merge my integration branch before release, so no need to worry at this stage.

Reply:
I did this as I saw you already had a branch with that version in flight ( think I've reviewed it now ) So I didn't want to have to go back in and change it when you merged yours 😄